src/components/PreFight/pre-fight.tsx

```diff
@@ -0,0 +1,29 @@
+import Champion from "../Arena/chamion";
+
+const PreFight = ({
+  className = "",
+  imageUrl,
+  champion1Url,
+  champion2Url,
+}: {
+  className?: string;
+  imageUrl: string;
+  champion1Url: string;
+  champion2Url: string;
+}) => {
+  return (
+    <div
+      className={`w-full bg-cover bg-center relative ${className}`}
+      style={{ backgroundImage: `url('${imageUrl}')` }}
```

Comment:
use tailwind built in `bg` class [example](https://tailwindcss.com/docs/background-image#basic-example)
something like:
```suggestion
      className=`bg-[url($imageUrl)]`
```

Reply:
Hi, I've looked into the other changes and they look good.
I want to approve this pr but I saw this change was missing🙂